backends/p4tools/modules/testgen/targets/bmv2/p4_asserts_parser.cpp

```diff
@@ -530,7 +530,7 @@ const IR::Node *AssertsParser::postorder(IR::P4Table *node) {
         auto *cloneProperties = node->properties->clone();
         IR::IndexedVector<IR::Property> properties;
         for (const auto *property : cloneProperties->properties) {
-            if (property->name.name != "key" || property->name.name != "entries") {
+            if (property->name.name != "key" && property->name.name != "entries") {
```

Comment:
@fruffy, note that the original condition was a tautology. I've changed it to match the comment above.

Reply:
Thanks for catching this. Hope I can get rid of this pass soon.